### PR TITLE
Replace old color variables with new Clumit colors

### LINKED ISSUE #181: Clumit light mode screen development (1) - Event, Detection, Triage, AI mode
Replace the color codes used on each of the following screens with a CSS variables

- Event
- Detection
- Triage
- AI Model

### LINKED ISSUE #182: Clumit light mode screen development (2) - Etc.
Replace color codes used on screens other than the listed below with CSS variables: Event, Detection, Triage, AI Model

ex) Sign In, Home, Dashboard, Report, Admin, K-MITRE, etc.

### LINKED ISSUE #185: Replace old variable names with new color names
```
    --bg-color: #1A1D20;
    --form-color: #212428;
    --title-text-color: #FFFFFF;
    --input-bg-color: rgba(97, 105, 116, 0.24);
    --submit-button-bg-color: #156EF2;
    --submit-button-bg-disabled-color: #156EF2;
    --submit-button-bg-hover-color: 0px 2px 8px 0px rgba(35, 125, 245, 0.40);
    --submit-button-opacity: 0.3;
    --alert-text-color: #F94C35;
    --menu-hover-color: #FFFFFF;
    --clumit-first-menu-default-bg-color: #212428;
    --search-condition-border-color: #2F333A;
    --dark-70: #2F333A;
    --transparent-mid-grey: rgba(97, 105, 116, 0.24);
    --transparent-dark-grey: rgba(97, 105, 116, 0.48);
    --transparent-light-grey: rgba(97, 105, 116, 0.08);
```

The above color names have been removed, but some are still used in our source code. We need to replace those names with the new color names.

### PR DESCRIPTION
**Pull request by jinjoolee07**
close #185
close #181
close #182

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Adjusted Clumit notification component's styling for consistency with the
   design.
+- Replaced deprecated color variable names with new color naming conventions.
```
Comment (sophie-cluml) [inline CHANGELOG.md:14]: Could you mention that it is related to Clumit?

**Comment by jinjoolee07** [comment]
> Could you please check the use of "Fixes" in the description?

Thank you for catching that. This has been addressed as well.

**Comment by sophie-cluml** [comment]
Could you rebase please? I think https://github.com/aicers/frontary/pull/190 used old variables so I think that needs to be dealt as well.

**Comment by jinjoolee07** [comment]
Rebase completed and updated the colors for [#190](https://github.com/aicers/frontary/pull/190). Let me know if there's anything more needed!

**Review by sophie-cluml — DISMISSED** [review]

**Comment by sophie-cluml** [comment]
@jinjoolee07 I found that the commit message needs to be corrected.

**Comment by jinjoolee07** [comment]
Thank you for pointing that out! I've now corrected the commit message.

**Review by sophie-cluml — APPROVED** [review]

**Pull request merged**